File: frontend/src/types/Workout.ts
export type Workout = {
  id: string
  trainingProgramId: string
  name: string
  dayOfWeek?: string | null
  notes?: string | null
  createdAt: string
  updatedAt: string
  description: string // Add this line
}

export type CreateWorkoutInput = {
  name: string
  dayOfWeek?: string
  notes?: string
}

export type UpdateWorkoutInput = {
  name: string
  dayOfWeek?: string
  notes?: string
}

File: backend/TrainMateX.Domain/Entities/Workout.cs
public class Workout
{
    public Guid Id { get; set; }

    // FK to TrainingProgram
    public Guid TrainingProgramId { get; set; }
    public string Name { get; set; } = null!;
    public string? DayOfWeek { get; set; }
    public string? Notes { get; set; }
    public DateTime CreatedAt { get; set; }

    //Navigation
    public TrainingProgram TrainingProgram { get; set; } = null!;
}